public/index.css

```diff
@@ -95,6 +95,10 @@ nav ul {
 	margin-top: 0.5rem;
 }
 
+.content > section > :is(ul, ol) :has(kbd) {
```

Comment:
[`:has()` isn’t supported in Firefox](https://caniuse.com/css-has) just yet, so this is only a partial fix and this approach also results in this mix of line heights which I think doesn’t look amazing:

<img width="552" alt="image" src="https://user-images.githubusercontent.com/357379/225454909-52ada1d4-f9fc-4d5c-abb3-36216e6b047f.png">

I think I might suggest a multi-part solution. This PR on the left, my suggestion on the right: 
<img width="1783" alt="image" src="https://user-images.githubusercontent.com/357379/225455224-6f7ce0ea-42a2-4108-9a03-f6df6eda874d.png">

How to do it:

1. Bump up line-height everywhere to `1.75`. This is a small change and I actually kind of like the extra breathing room it gives.

2. Tweak `kbd` styles to take up slightly less vertical space:

   - `box-shadow: 0 2px var(--theme-shade-subtle)` — 2px instead of the current 3px
   
   - `padding: 0.0625rem 0.375rem` — 1px instead of 2px vertical padding

Reply:
I considered "everywhere" as in, content paragraphs & lists, which I liked, but bumping everywhere to 1.75 simply didn't feel right for some reason.